### PR TITLE
fix: update logo on documentation pages

### LINKED ISSUE #3545: The logo on the documentation pages is pixelated
**Describe the bug; what happened?**
The logo in the top left corner is pixelated on all documentation pages and also on component explorer site, both sites.

**If applicable, provide screenshots:**
Screen Shot 2020-07-23 at 4 44 25 PM

**In what environment did you see the issue?**

* OS & Device: [e.g. **MacOS**, iOS, Windows, Linux] on [iPhone 7, PC]
* Browser [e.g. **Edge**, Chrome, Safari, FireFox]
* Version [e.g. 85]

**Is there any additional context?**

### PR DESCRIPTION
**Pull request by JazibJafri**
# Description

I have used the svgs provided by @Falkicon. However I have not updated the `docusaurus.config.js` as it uses the logo from the CDN. @EisenbergEffect the `docusaurus.config.js` should work as is (if the new images are uploaded with the same name).
Let me know if you want anything else updated.

## Motivation & context

Fixes #3545
## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

**Comment by EisenbergEffect** [comment]
Thanks @JazibJafri for working on this.
@janechu I think we need your review on this as well as a review from @awentzel

**Comment by awentzel** [comment]
I pull down and tested, pushed up one change to switch Docusaurus to use the new SVGs.  All looks good.  Thanks for the contribution.

File: sites/fast-component-explorer/app/explorer.tsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ import {
 } from "./explorer.props";
 import { previewReady } from "./preview";
 /* eslint-disable-next-line @typescript-eslint/no-var-requires */
-const FASTInlineLogo = require("@microsoft/site-utilities/statics/assets/fast-inline-logo.png");
+const FASTInlineLogo = require("@microsoft/site-utilities/statics/assets/fast-inline-logo.svg");
```
Comment (janechu) [inline sites/fast-component-explorer/app/explorer.tsx:68]: you will need to add "svg" to the webpack.config.js for this site, change the file-loader test to:

```javascript
{
    test: /\.(svg|png|jpe?g|gif)$/i,
    use: {
        loader: "file-loader",
        options: {
            esModule: false,
        },
    },
},
```

File: sites/fast-creator/app/creator.tsx [hunk 2]
```diff
@@ -68,7 +68,7 @@ import { ProjectFileTransfer } from "./components";
 import { selectDeviceOverrideStyles } from "./utilities/style-overrides";
 import { previewReady } from "./preview";
 /* eslint-disable-next-line @typescript-eslint/no-var-requires */
-const FASTInlineLogo = require("@microsoft/site-utilities/statics/assets/fast-inline-logo.png");
+const FASTInlineLogo = require("@microsoft/site-utilities/statics/assets/fast-inline-logo.svg");
```
Comment (janechu) [inline sites/fast-creator/app/creator.tsx:71]: Same situation as above, just add the "svg" to the file-loader in the webpack.config.js for this site

File: sites/fast-tooling-examples/app/index.ts [hunk 3]
```diff
@@ -15,7 +15,7 @@ import examples from "./registry";
 import toolingGuidance from "./.tmp/tooling-guidance";
 import toolingReactGuidance from "./.tmp/tooling-react-guidance";
 /* eslint-disable-next-line @typescript-eslint/no-var-requires */
-const FASTInlineLogo = require("@microsoft/site-utilities/statics/assets/fast-inline-logo.png");
+const FASTInlineLogo = require("@microsoft/site-utilities/statics/assets/fast-inline-logo.svg");
```
Comment (janechu) [inline sites/fast-tooling-examples/app/index.ts:18]: Here as well, see previous comment about adding to the webpack.config.js to include "svg"

File: sites/fast-component-explorer/app/explorer.tsx (hunk 1 above)
Comment (JazibJafri) [reply sites/fast-component-explorer/app/explorer.tsx:68]: @janechu okay, will do.

**Comment by JazibJafri** [comment]
@janechu updated the webpack configs.

**Comment by EisenbergEffect** [comment]
Thanks @JazibJafri !

**Pull request merged**